docs/development/core/server/kibana-plugin-server.getauthstate.md

```diff
@@ -4,13 +4,13 @@
 
 ## GetAuthState type
 
-Get authentication state for a request. Returned by `auth` interceptor.
+Gets authentication state for a request. Returned by `auth` interceptor.
 
 <b>Signature:</b>
 
 ```typescript
-export declare type GetAuthState = (request: KibanaRequest | LegacyRequest) => {
+export declare type GetAuthState = <T = unknown>(request: KibanaRequest | LegacyRequest) => {

```

Comment:
Should we just expose this on `RequestHandlerContext`?

Reply:
We can expose `auth.isAuthenticated` when optional authentication implemented. Not sure we need to provide `auth.get` until then or can be exposed later upon request.